client/ayon_shotgrid/plugins/publish/validate_shotgrid_user.py

```diff
@@ -20,28 +20,29 @@ def process(self, context):
             raise PublishValidationError("Missing Shotgrid Credentials")
 
         self.log.info("Login Shotgrid set in Ayon is {}".format(user_login))
-        self.log.info("Current shotgun Project is {}".format(sg_project))
+        self.log.info("Current Shotgrid Project is {}".format(sg_project))
 
         sg_user = sg_session.find_one(
             "HumanUser",
             [
                 ["login", "is", user_login],
                 ["projects", "name_contains", project_name]
             ],
-            ["projects"]
+            ["projects", "permission_rule_set"]
         )
+        admin = sg_user["permission_rule_set"]["name"] == "Admin"
```

Comment:
This probably should be something like:
`admin = True if sg_user["permission_rule_set"]["name"] == "Admin" else False`

But I believe that line 29 might prevent from an admin  to be shown

Reply:
Yeah that's more explicit and clear, I will update it to your snippet, thank you!

Reply:
done here https://github.com/ynput/ayon-shotgrid/pull/58/commits/9bc1af93b2c5713b961cd7a776924640cd80a06e